multiversx_sdk_cli/dependencies/modules.py

```diff
@@ -253,7 +253,24 @@ def is_installed(self, tag: str) -> bool:
         logger.info(f"which twiggy: {which_twiggy}")
 
         dependencies = [which_rustc, which_cargo, which_sc_meta, which_wasm_opt, which_twiggy]
-        return all(dependency is not None for dependency in dependencies)
+        installed = all(dependency is not None for dependency in dependencies)
+
+        if installed:
+            actual_version_installed = self._get_actual_installed_version()
+
+            if tag in actual_version_installed:
+                logger.info(f"[{self.key} {tag}] is installed.")
+            elif "Command 'rustup' not found" in actual_version_installed:
```

Comment:
The error message might look differently on MacOS. E.g.

```
multiversx@papple ~ % rustupfoobar
zsh: command not found: rustupfoobar
multiversx@papple ~ % 
```

Thus, we can check for `"not found" in actual_version_installed` perhaps?

Reply:
Indeed. Now the `elif` statement is as above.